trl/integration_utils.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
```

Comment:
I think it's fine having this function directly in `utils.py`.

Reply:
Done